src/System.Diagnostics.EventLog/tests/EventLogTests/EventLogTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    public class EventLogTests
+    {
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void EventLogReinitializationException()
+        {
+            using (EventLog eventLog = new EventLog())
+            {
+                eventLog.BeginInit();
+                Assert.Throws<InvalidOperationException>(() => eventLog.BeginInit());
+                eventLog.Close();
```

Comment:
EventLog.Close() actually doesn't do anything that EventLog.Dispose() doesn't do (check out the code)
So you can remove this line, and let the `using` statement do the Dispose.